rfcs/033-lineage-graph.md

```diff
@@ -0,0 +1,138 @@
+- Feature Name: lineage-graph-visualization
+- Start Date: 2021-04-01
+- RFC PR: [amundsen-io/rfcs#33](https://github.com/amundsen-io/rfcs/pull/33)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Amundsen Lineage Stage 1 - Graph Visualization
+
+## Summary
+
+This is the next in the series of https://github.com/amundsen-io/rfcs/pull/25
+
+Implement a navigable canvas-based graph visualization for lineage. Lineage is traditionally represented this way, suboptimally, in the various existing tools like CompilerWorks, Castor, Apache Atlas, etc.
```

Comment:
is there any research explaining shortcomings of existing tools? would be good to understand on which of them we are trying to improve. for one, I think Apache Atlas lineage view is actually pretty good

Reply:
Atlas lineage is pretty extensive and is good. I agree. And we can improve on these things based on the user feedback. Controlling the depth and clustering the nodes is the advantage over the other tools i.e., showing top X nodes, etc.